### PR TITLE
Hotfix/measurement

### PR DESCRIPTION
**Pull request by lkosanke**

### CODE REVIEW

**Comment by lkosanke** [comment]
Sovles #121 and 
https://github.com/formal-methods-mpi/Literature-Research-Pilot/issues/60

Should I add tests as well?

**Comment by lkosanke** [comment]
Also solves this:
https://github.com/formal-methods-mpi/Literature-Research-Pilot/issues/59

File: src/taxons/measurement.jl [hunk 1]
```diff
@@ -50,7 +53,17 @@ function Measurement(j...;
...
+    quest_scale=missing)
+
+    # Convert quest_scale to JudgementInt or JudgementNumber if necessary
+    quest_scale_converted = if isa(quest_scale, Int)
+                                JudgementInt(quest_scale)  # Convert Int to JudgementInt
+                            elseif isa(quest_scale, Float64)
+                                JudgementNumber(quest_scale)  # Convert Float64 to JudgementNumber
+                            else
+                                quest_scale  # Keep as is (including Missing)
+                            end
```
Comment (aaronpeikert) [inline src/taxons/measurement.jl:66]: This part deserves a test.

**Review by aaronpeikert — APPROVED** [review]

File: src/taxons/measurement.jl (hunk 1 above)
Comment (lkosanke) [reply src/taxons/measurement.jl:66]: Test as use case of function

**Pull request merged**